spec/models/document_spec.rb

```diff
@@ -33,12 +33,13 @@ def initialize(params = {})
             :base_path,
             :content_id,
             :updated_at,
```

Comment:
Do we use `updated_at` for anything or can this be removed from the list of fields we request?

If so, we can also remove it from document's fields and make things simpler.